CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### HEAD
 * Update to Bootstrap 3.3.4
+* Fix `gulp --production` race condition. ([1398](https://github.com/roots/sage/issues/1398))
```

Comment:
this needs to match the rest of the changelog, so remove the period and add a `#` before the issue number :mag: